filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1900,11 +1900,12 @@ bool OpenGLDriver::isSRGBSwapChainSupported() {
 }
 
 bool OpenGLDriver::isStereoSupported() {
-    // Stereo requires instancing and EXT_clip_cull_distance.
+    // Instanced-stereo requires instancing and EXT_clip_cull_distance.
+    // Multiview-stereo requires ES 3.0 and OVR_multiview2.
     if (UTILS_UNLIKELY(mContext.isES2())) {
         return false;
     }
-    return mContext.ext.EXT_clip_cull_distance;
+    return mContext.ext.EXT_clip_cull_distance || mContext.ext.OVR_multiview2;
```

Comment:
I'm not quite satisfied with this as it could mean only one of them is available. If we can use a preprocessor, something like BACKEND_OPENGL_MULTIVIEW_STEREO and BACKEND_OPENGL_INSTANCED_STEREO, the semantics of this function will become more specific. Also it does mean the engine only has one algorithm baked at compile time, which I preferred.

Reply:
I think Filament will want the ability to query what type(s) of stereo each backend supports. Could we change the signature to `bool OpenGLDriver::isStereoSupported(StereoType type)`?

Reply:
I update the changes as suggested.